components/tyk-dashboard/templates/deployment-dashboard.yaml

```diff
@@ -0,0 +1,255 @@
+        traffic.sidecar.istio.io/includeInboundPorts: ""
+      {{- end }}
+        checksum/secrets: {{ include (print $.Template.BasePath "/secrets.yaml") . | sha256sum }}
+        {{- if .Values.mounts }}
+        {{- range $secret := .Values.mounts }}
+        "checksum/secret-{{ $secret.name }}": {{ include (print "mounts/%s" $secret.filename) . | sha256sum }}
+        {{- end }}
+        {{- end }}
+    spec:
+{{- if .Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.nodeSelector | indent 8 }}
+{{- end }}
+{{- if .Values.tolerations }}
+      tolerations:
+{{ toYaml .Values.tolerations | indent 8 }}
+{{- end }}
+{{- if .Values.affinity }}
+      affinity:
+{{ toYaml .Values.affinity | indent 8 }}
+{{- end }}
+      containers:
+      - image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        name: dashboard-{{ .Chart.Name }}
+        securityContext:
+          runAsUser: 999
+          runAsGroup: 999
+{{/*          fsGroup: 2000*/}}
+        env:
+          ## DEFAULT TYK DASHBOARD CONFIGURATION
+          - name: "TYK_DB_ENABLEOWNERSHIP"
+            value: "{{ .Values.config.enableOwnership }}"
+          - name: "TYK_DB_PAGESIZE"
+            value: "{{ .Values.config.defaultPageSize }}"
+          - name: "TYK_DB_REDISPORT"
+            value: "{{ .Values.redis.port }}"
+          - name: "TYK_DB_REDISHOST"
+            value: "{{ .Values.redis.host }}"
+          - name: "TYK_DB_NOTIFYONCHANGE"
+            value: "{{ .Values.config.notifyOnChange }}"
+          - name: "TYK_DB_HASHKEYS"
+            value: "{{ .Values.config.hashKeys }}"
+          - name: "TYK_DB_ENABLEDUPLICATESLUGS"
+            value: "{{ .Values.config.enableDuplicateSlugs }}"
+          - name: "TYK_DB_SHOWORGID"
+            value: "{{ .Values.config.showOrgId }}"
+          - name: "TYK_DB_ENABLEHOSTNAMES"
+            value: "{{ .Values.config.hostConfig.enableHostNames }}"
+          - name: "TYK_DB_DISABLEORIGSLUGPREFIX"
+            value: "{{ .Values.config.hostConfig.disableOrgSlugPrefix }}"
+          - name: "TYK_DB_OVERRIDEHOSTNAME"
+            value: "{{ .Values.config.hostConfig.overrideHostname }}"
+          - name: "TYK_DB_HOMEDIR"
+            value: "{{ .Values.config.homeDir }}"
+          - name: "TYK_DB_USESHARDEDANALYTICS"
+            value: "{{ .Values.config.useShardedAnalytics }}"
+          - name: "TYK_DB_ENABLEAGGREGATELOOKUPS"
+            value: "{{ .Values.config.enableAggregateLookups }}"
+          - name: "TYK_DB_ENABLEANALYTICSCACHE"
+            value: "{{ .Values.config.enableAnalyticsCache }}"
+          - name: "TYK_DB_ALLOWEXPLICITPOLICYID"
+            value: "{{ .Values.config.allowExplicitPolicyId }}"
+          - name: "TYK_DB_OAUTHREDIRECTURISEPARATOR"
+            value: "{{ .Values.config.oauthRedirectUriSeparator }}"
+          - name: "TYK_DB_KEYREQUESTFIELDS"
```

Comment:
i couldn't find this configuration on the docs. is it correct?

Reply:
Solved